weathervane.config.docker

```diff
@@ -0,0 +1,60 @@
+{
+  # This is a temporary version of a docker config file.
+  # We can't use the fixed config sizes because the auctionBidServer does
+  # run in Docker-only yet.
```

Comment:
typo?: "does run in Docker-only yet" -> "does not run in Docker yet" 
Here and below.

Reply:
Thanks.  Fixing in a new commit